api-reference/10 UI Widgets/dxPopover/1 Configuration/position.md

```diff
@@ -16,4 +16,4 @@ Besides the position configuration object, the option can take on the following
 - "left" - places the popover to the left of the target element
 - "right" - places the popover to the right of the target element
 
-[note]To specify the element against which the widget will be positioned, use the [target](/api-reference/10%20UI%20Widgets/dxPopover/1%20Configuration/target.md '{basewidgetpath}/Configuration/#target') option instead of the [of](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') field of the **position** object. If both **target** and **of** are specified, **target** will be used.
\ No newline at end of file
+You can use the [position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') and the [target](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#target) options to specify the element against which the **Popover** will be positioned. If both **target** and **position**.**of** are specified, the latter will take precedence. However, if the **of** field of the **position** object is not specified, the **target** option will be used to define the **Popover**'s position.
```

Comment:
1. This paragraph should go before the list because it discusses a field from the `position` object, while the list starts with "besides the position object".
2. It's unclear that the `target` option belongs to the Popover, not to the `position` object.
3. The link attached to the word "position" leads to this same article.
4. The last two sentences can be significantly shorter: 

> If both these options are specified, **position**.**of** takes precedence.

Let's get rid of passive voice:

> If you set both these options, **position**.**of** takes precedence.